single_subject/batch_single_subject.sh

```diff
@@ -146,10 +147,33 @@ fsleyes t2.nii.gz -cm greyscale t2_seg.nii.gz -cm subcortical -a 70.0 &
 # Registering T2 data to the PAM50 template
 # ======================================================================================================================
 cd ../t2
+
+# Create labels at C3 and T2 mid-vertebral levels. These labels are needed for template registration.
+sct_label_utils -i t2_seg_labeled.nii.gz -vert-body 3,9 -o t2_labels_vert.nii.gz
+# Generate a QC report to visualize the two selected labels on the anatomical image
+sct_qc -i t2.nii.gz -s t2_labels_vert.nii.gz -p sct_label_utils -qc ~/qc_singleSubj
+
+# OPTIONAL: You might want to completely bypass sct_label_vertebrae and do the labeling manually. In that case, we
+# provide a viewer to do so conveniently. In the example command below, we will create labels at the inter-vertebral
+# discs C2-C3 (value=3), C3-C4 (value=4) and C4-C5 (value=5).
+#sct_label_utils -i t2.nii.gz -create-viewer 3,4,5 -o labels_disc.nii.gz -msg "Place labels at the posterior tip of each inter-vertebral disc. E.g. Label 3: C2/C3, Label 4: C3/C4, etc."
+
 # Register t2->template.
 sct_register_to_template -i t2.nii.gz -s t2_seg.nii.gz -l t2_labels_vert.nii.gz -c t2 -qc ~/qc_singleSubj
 # Note: By default the PAM50 template is selected. You can also select your own template using flag -t.
 
```

Comment:
In the slides the PAM50 registration has been moved to be before Lesion analysis. This would move lines 152-187 to line 95; it would also remove the need to re-enter the T2 directory. Should we do this too>

Reply:
I think we still require re-entering the T2 directory due to the move to the `t2_compression` folder as part of the shape metric section?